brianlong/www.validators.app
# 182298198 tweak views and design

#### What's this PR do?
- This story tweaks html views: adds missing classes, removes unnecessary classes and tags, etc. Fixes headers structure. Also refreshes design a little bit.

#### How should this be manually tested?
- Click through the app and make sure it looks good, no 500 etc.
- On most pages you should see no changes (or almost no changes) comparing to production version - I only tweaked html structure in most places

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/n/projects/2177859/stories/182298198)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
